package.json

```diff
@@ -22,7 +22,7 @@
   },
   "homepage": "https://github.com/moneyadviceservice/isomorphic_react_prototype#readme",
   "dependencies": {
-    "@moneypensionservice/directories": "^1.5.0",
+    "@moneypensionservice/directories": "^1.6.1",
```

Comment:
Due to changes on the `Header` on v1.6.1 you now need to change the prop `currentLgn`  to `currentLng` and provide a translation object to the `Header` component as well, with similar structure to the json files on the library.
Because of this, currently the translations are not working for the `Header`.

Reply:
Done. kindly let me know ahead of time the details of changes made to existing components. Thanks